packages/contracts/contracts/L1/Colosseum.sol

```diff
@@ -1107,31 +1114,6 @@ contract Colosseum is Initializable, ISemver {
         return _challengeStatus(challenges[_outputIndex][_challenger]);
     }
 
-    /**
-     * @notice Returns the segment length required for that turn.
-     *
-     * @param _turn The challenge turn.
-     *
-     * @return The segments length.
-     */
-    function getSegmentsLength(uint8 _turn) public view returns (uint256) {
-        if (_turn < TURN_INIT) revert InvalidTurn();
```

Comment:
This condition prevents invalid `turn` parameter. Is there no such case?

Reply:
I checked all the logics which utilize this function, there was no such case.